.github/workflows/release.yml

```
@@ -22,68 +23,89 @@ jobs:
 
             - name: Build cross-platform binaries
-              run: |
-                  mkdir -p binaries
-                  bun build src/index.ts --compile --target=bun-linux-x64 --outfile binaries/dbmux-linux-x64
-                  bun build src/index.ts --compile --target=bun-darwin-x64 --outfile binaries/dbmux-darwin-x64
-                  bun build src/index.ts --compile --target=bun-darwin-arm64 --outfile binaries/dbmux-darwin-arm64
-                  bun build src/index.ts --compile --target=bun-windows-x64 --outfile binaries/dbmux-windows-x64.exe
+              run: bun run build:binaries
 
             - name: Create checksums
               run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2035:info:2:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)